cal/api/serializers.py

```diff
@@ -57,22 +59,25 @@ class Meta:
         fields = ('name', 'start_time', 'end_time')
 
 
-class ColorCategorySerializer(serializers.ModelSerializer):
+class CategorySerializer(serializers.ModelSerializer):
 
     hours = serializers.SerializerMethodField()
     category_color = serializers.SerializerMethodField()
 
     def get_fields(self, *args, **kwargs):
-        fields = super(ColorCategorySerializer, self).get_fields(*args, **kwargs)
+        fields = super(CategorySerializer, self).get_fields(*args, **kwargs)
         fields['calendar'].queryset = fields['calendar'].queryset.filter(user=self.context['request'].user)
         return fields
 
     class Meta:
-        model = ColorCategory
+        model = Category
         fields = ('id', 'label', 'hours', 'calendar', 'category_color')
 
     def get_hours(self, obj):
-        calendar_ids = self.context['calendar_ids']
+        if self.context.get('calendar_ids') is not None:
+            calendar_ids = ast.literal_eval(self.context['calendar_ids'])
```

Comment:
What does this do? ast.literal_eval? Abstract syntax trees don't really make sense to me, maybe add a comment?